gradio/imagetools.py

```diff
@@ -0,0 +1,50 @@
+from typing import Literal
```

Comment:
add `from __future__ import annotations` or else we can't import in 3.8 because of `|`